common/gtk-2.0/main.rc

```diff
@@ -1949,13 +1949,11 @@ style "progressbar" {
 }
 
 style "separator_menu_item" {
-
   engine "pixmap" {
-
     image {
       function = BOX
-      file = "assets/null.png"
-      border = { 0, 0, 1, 0 }
+      file = "assets/line-h.png"
+      border = { 20, 0, 3, 0 }
```

Comment:
Just to understand your changes:
You made the actual space between two sections smaller, but added a horizontal line for it instead, right?

Could you please explain the values 20,0,3,0 in short? (I am also learning gtk, would be nice if you can tell me why you've chosen those).

Now I am wondering if horst did the extra space (instead of lines) by intention. However from your screenshot it looks okay for the light theme, but we need to test it for the dark theme as well. Can you please test this and also add a screenshot for darker and dark variant?

Reply:
I have no clue why 20,0,3,0, but it just works